docs/source/routing/cloud/configuration.mdx

```diff
@@ -234,43 +234,45 @@ For a more general introduction to CORS and common considerations, see the follo
+- Add the origins of those web applications to the router's list of allowed `policies`.
   - Use this option if there is a known, finite list of web applications that consume your cloud supergraph.
-- Add a regular expression that matches the origins of those web applications to the router's list of allowed `origins`.
+- Add a regular expression that matches the origins of those web applications to the router's list of allowed `policies`.
   - This option comes in handy if you want to match origins against a pattern, see the example below that matches subdomains of a specific namespace.
 - Enable the `allow_any_origin` option.
-
   - Use this option if your supergraph is a public API with arbitrarily many web app consumers.
   - With this option enabled, the router sends the [wildcard (`*`)](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Origin#directives) value for the `Access-Control-Allow-Origin` header. This enables any website to initiate browser connections to it (but they can't provide cookies or other credentials).
 
-- You must use the `origins` + `match_origins` option if clients need to [authenticate their requests with cookies](#passing-credentials).
+- You must use policies with `origins` + `match_origins` if clients need to [authenticate their requests with cookies](#passing-credentials).
 
-The following snippet includes an example of each option (use either `allow_any_origin`, or `origins + match_origins`):
+The following snippet includes an example of each option (use either `allow_any_origin`, or `policies` with specific origins):
 
 ```yaml
 cors:
   # Set to true to allow any origin
   # (Defaults to false)
   allow_any_origin: true
 
-  # List of accepted origins
+  # List of CORS policies with specific origins and settings
   # (Ignored if allow_any_origin is true)
-  # (Defaults to the GraphOS Studio url: `https://studio.apollographql.com`)
+  # (Defaults to allowing GraphOS Studio: `https://studio.apollographql.com`)
   #
   # An origin is a combination of scheme, hostname and port.
   # It does not have any path section, so no trailing slash.
-  origins:
-    - https://www.your-app.example.com
-    - https://studio.apollographql.com # Keep this so GraphOS Studio can run queries against your router
-  match_origins:
-    - 'https://([a-z0-9]+[.])*api[.]example[.]com' # any host that uses https and ends with .api.example.com
+  policies:
+    - origins:
+        - https://www.your-app.example.com
+        - https://studio.apollographql.com # Keep this so GraphOS Studio can run queries against your router
+    - origins:
+        - https://api.example.com
+      match_origins:
+        - 'https://([a-z0-9]+[.])*api[.]example[.]com' # any host that uses https and ends with .api.example.com
 ```
 
-You can also turn off CORS entirely by setting `origins` to an empty list:
+You can also disable CORS entirely by setting `policies` to an empty list:
```

Comment:
I would suggest a different wording here - both "turn off" and "disable" might cause inexperienced devs to think "if I do this, I don't have to deal with CORS anymore", where in reality, CORS itself cannot be disabled as it's a browser security feature.


```suggestion
You can also forbid CORS requests entirely by setting `policies` to an empty list:
```